packages/lisk-p2p/src/helpers.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { VError } from 'verror';
+import { Peer } from './peer';
+
+export interface PeerReturnType {
+	readonly options: PeerOptions;
```

Comment:
in core, I think it's returning calculated network height here.
options would be misleading, and returning the input doesn't make much sense.
we don't need to follow core exactly, and maybe network height where peers are selected from might be useful information.